registry/auth/token/util.go

```diff
@@ -78,6 +79,10 @@ func GetRFC7638Thumbprint(publickey crypto.PublicKey) string {
 		y := base64.RawURLEncoding.EncodeToString(params.Gy.Bytes())
 
 		payload = fmt.Sprintf(`{"crv":"%s","kty":"EC","x":"%s","y":"%s"}`, crv, x, y)
+	case ed25519.PublicKey:
```

Comment:
Not sure, but I see all the other branches check for a pointer-type, should that be the same here as well?

```suggestion
	case *ed25519.PublicKey:
```

Reply:
`ed25519.PublicKey` is defined as `type PublicKey []byte`, which makes it suitable to be passed by-value.
Its implementation of the `crypto.PublicKey` interface is defined on non-pointer receiver, `func (pub PublicKey) Equal(x crypto.PublicKey) bool`.

Reply:
Ah, thanks! I commented from my phone after posting 😂